tribits/common_tpls/FindTPLNetcdf.cmake

```diff
@@ -44,98 +44,158 @@
 # in order to trigger the right behavior in the function
 # tribits_tpl_find_include_dirs_and_libraries().
 #
-
-set(REQUIRED_HEADERS netcdf.h)
-set(REQUIRED_LIBS_NAMES netcdf)
-
-if (TPL_ENABLE_MPI)
-  set(REQUIRED_LIBS_NAMES ${REQUIRED_LIBS_NAMES} pnetcdf)
+if (${CMAKE_VERSION} GREATER "3.13")
```

Comment:
```cmake
if(NOT CMAKE_VERSION VERSION_LESS "3.13")
```

Though, you're now requiring CMake 3.17 or so right? So is this even needed?

Reply:
That is a good point.  This file only gets pulled in for TriBITS projects where CMake 3.17+ is required (and soon will be 3.22+).

Reply:
This is resolved in the new commit [`4127bf3` (#535)](https://github.com/TriBITSPub/TriBITS/pull/535/commits/4127bf3e4cda37614da92489ce7dba643dfabf45).